.eslintrc.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
   env: {
     node: true,
   },
-  extends: ['plugin:vue/essential', '@vue/airbnb', '@vue/typescript'],
+  extends: ['plugin:vue/essential', '@vue/airbnb', '@vue/typescript', 'prettier'],
```

Comment:
Hm, I've been adding this line since Prettier doesn't seem to work for me otherwise